apps/commudle-admin/src/app/feature-modules/dashboard/components/user-dashboard/user-dashboard.component.ts

```diff
@@ -0,0 +1,69 @@
+import { Component, OnInit } from '@angular/core';
+import { EDbModels, ICommunity, IPageInfo } from '@commudle/shared-models';
```

Comment:
I think we can combine two paths 
import { EDbModels, ICommunity, IPageInfo } from '@commudle/shared-models'; and 
import { IUserStat } from 'libs/shared/models/src/lib/user-stats.model';
because @commudle/shared-models means ibs/shared/models/src/lib/user-stats.mode